dhis-2/dhis-api/src/main/java/org/hisp/dhis/category/CategoryOptionCombo.java

```diff
@@ -217,7 +218,7 @@ public boolean isDefault()
      */
     public DateRange getDateRange( DataSet dataSet )
     {
-        return getDateRange( dataSet, null );
+        return getDateRange( dataSet, null, null );
```

Comment:
I would recommend using overloaded versions of "getDateRange" in such cases, to avoid passing null arguments in different positions.

Reply:
I agree with Maikel here :)